config/linux_gcc.cmake

```diff
@@ -17,6 +22,12 @@ if(NOT __processedUserDefaults)
   # Note (12/2021): passing -march=native to gfortran seems to slow down the
   # PPM solver
 
+  set(__ARCH_C_OPT_FLAGS "-O3 -g -funroll-loops")
+  set(CMAKE_C_FLAGS_RELEASE "${__ARCH_C_OPT_FLAGS}")
+  set(CMAKE_C_FLAGS_RELWITHDEBINFO "-g ${__ARCH_C_OPT_FLAGS}")
+  set(CMAKE_CXX_FLAGS_RELEASE "${__ARCH_C_OPT_FLAGS}")
+  set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g ${__ARCH_C_OPT_FLAGS}")
+
```

Comment:
```suggestion
```

I think these are now all unnecessary due to some more recent changes.

If you look further down on the page, we define `ENZOE_C_FLIST_INIT` and `ENZOE_CXX_FLIST_INIT`. 
```c++
  set(ENZOE_C_FLIST_INIT "-Wall;-funroll-loops")
  set(ENZOE_CXX_FLIST_INIT "${ENZOE_C_FLIST}")
```
Currently, these add the `-Wall` and `-funroll-loops` flags. If you also want the `-O3` flag instead of `-O2`, you should modify the list so that it reads `-Wall;-funroll-loops;-O3`.

As an aside, we shouldn't be adding `-g` as a symbol here. The canonical way to get debugger symbols is to use `-DCMAKE_BUILD_TYPE=RelWithDebInfo` when configuring the build-system.